tldr.py

```diff
@@ -241,32 +243,34 @@ def get_page(
         languages = get_language_list()
     # only use cache
     if USE_CACHE:
+        for remote in sources:
+            for platform in platforms:
+                for language in languages:
+                    if platform is None:
+                        continue
+                    try:
+                        return get_page_for_platform(
+                            command,
+                            platform,
+                            remote,
+                            language,
+                            only_use_cache=True,
+                        )
+                    except CacheNotExist:
+                        continue
+    for remote in sources:
         for platform in platforms:
             for language in languages:
                 if platform is None:
                     continue
                 try:
-                    return get_page_for_platform(
-                        command,
-                        platform,
-                        remote,
-                        language,
-                        only_use_cache=True,
-                    )
-                except CacheNotExist:
-                    continue
-    for platform in platforms:
-        for language in languages:
-            if platform is None:
-                continue
-            try:
-                return get_page_for_platform(command, platform, remote, language)
-            except HTTPError as err:
-                if err.code != 404:
-                    raise
-            except URLError:
-                if not PAGES_SOURCE_LOCATION.startswith('file://'):
-                    raise
+                    return get_page_for_platform(command, platform, remote, language)
+                except HTTPError as err:
+                    if err.code != 404:
+                        raise
+                except URLError:
+                    if not remote.startswith('file://'):
+                        raise
```

Comment:
Unfortunately this looks scarier than the actual change. The main addition here is the additional loop due to multiple sources.

Removing the global variable reference also subtracts from the diff clarity (line 268 of the removal, 272 of the addition)